src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -5808,4 +5808,32 @@ elementFormDefault="qualified">
     </xs:complexType>
   </xs:element>
 
+  <xs:element name="InternalsVisibleTo" substitutionGroup="msb:Item">
+  <xs:annotation>
+    <xs:documentation>
+      <!-- _locID_text="InternalsVisibleTo" _locComment="" -->Specifies that types that are ordinarily visible only within the assembly are visible to the specified assemblies.
```

Comment:
and methods?

Reply:
I thought the exact same thing when I copied that text from the docs. It should likely be "types and members" to be even more correct. I'll give the wording a massage with the goal of just making it clearer here despite what the API docs have right now.